salesagility/SuiteCRM
# Format a lot of minified JavaScript

## Description
The code was seemingly minified before, which made it essentially unreadable. This formats it with Prettier since it's a commonly-used formatting tool and didn't require much manual work from me.

## How To Test This
Make sure the FP Events module still works.

You can also run the original code through Prettier yourself to verify that the changes are the same as mine in this PR. I used the [Prettier Playground](https://prettier.io/playground/) and set the `--print-width` flag to 200 to prevent prettier from wrapping a bunch of code in weird spots.
I formatted the following files:

- [`modules/calendar/Cal.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/calendar/Cal.js)
- [`modules/ACLRoles/ACLRoles.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/ACLRoles/ACLRoles.js)
- [`modules/AM_ProjectTemplates/jsclass_scheduler.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/AM_ProjectTemplates/jsclass_scheduler.js)
- [`modules/Campaigns/DetailView.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Campaigns/DetailView.js)
- [`modules/Campaigns/WebToLead.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Campaigns/WebToLead.js)
- [`modules/Campaigns/wizard.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Campaigns/wizard.js)
- [`modules/Connectors/Connector.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Connectors/Connector.js)
- [`modules/Contacts/Contact.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Contacts/Contact.js)
- [`modules/Documents/documents.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Documents/documents.js)
- [`modules/EAPM/EAPMEdit.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/EAPM/EAPMEdit.js)
- [`modules/EmailTemplates/EmailTemplate.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/EmailTemplates/EmailTemplate.js)
- [`modules/FP_events/duration_dependency.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/FP_events/duration_dependency.js`)
- [`modules/FP_events/jsclass_scheduler.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/FP_events/jsclass_scheduler.js`)
- [`modules/MergeRecords/Merge.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/MergeRecords/Merge.js)
- [`modules/Leads/Lead.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Leads/Lead.js)
- [`modules/InboundEmail/InboundEmail.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/InboundEmail/InboundEmail.js)
- [`modules/Home/tour.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Home/tour.js)
- [`modules/Schedulers/Schedulers.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Schedulers/Schedulers.js)
- [`modules/Studio/studio.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Studio/studio.js)
- [`modules/Studio/studiodd.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Studio/studiodd.js)
- [`modules/Studio/studiotabgroups.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Studio/studiotabgroups.js)
- [`modules/Studio/ygDDListStudio.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Studio/ygDDListStudio.js)
- [`modules/UpgradeWizard/upgradeWizard.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/UpgradeWizard/upgradeWizard.js)
- [`modules/Users/DetailView.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Users/DetailView.js) (#7332)
- [`modules/Users/login.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Users/login.js)
- [`modules/Users/PasswordRequirementBox.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Users/PasswordRequirementBox.js)
- [`modules/Users/UserEditView.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Users/UserEditView.js)
- [`modules/Users/User.js`](https://github.com/salesagility/SuiteCRM/blob/develop/modules/Users/User.js)
- [`service/utils/SugarRest.js`](https://github.com/salesagility/SuiteCRM/blob/develop/service/utils/SugarRest.js)

## Types of changes

### Final checklist
